captum/attr/_core/feature_ablation.py

```diff
@@ -284,6 +284,9 @@ def attribute(
 
             # Iterate through each feature tensor for ablation
             for i in range(len(inputs)):
+                # Skip any empty input tensors
+                if torch.numel(inputs[i]) == 0:
```

Comment:
For the empty input, can't the use pass it as an additional forward args since the attribution is empty too ?

Reply:
The issue is that this (sparse) tensor may not be empty for all batches, so to pass a consistent structure, it would be good to have this.

Reply:
I see,  makes sense